etcdserver/cluster_util.go

```diff
@@ -172,18 +226,27 @@ func getVersion(m *Member, tr *http.Transport) (string, error) {
 	for _, u := range m.PeerURLs {
 		resp, err = cc.Get(u + "/version")
 		if err != nil {
+			log.Printf("etcdserver: failed to reach the peerURL(%s) of member %s (%v)", u, m.ID, err)
```

Comment:
@barakmich

How about this? We actually log the per member error information at the caller side (line 129).
I added per endpoint error message.